feat(pushmsix):Add ESR_APPLICATION_ID for Thunderbird

stephenking314 opened the pull request.

stephenking314 committed:
feat(pushmsix):Add ESR_APPLICATION_ID for Thunderbird

hneiva commented on pushmsixscript/docker.d/worker.yml:
```diff
@@ -11,6 +11,7 @@ application_ids:
   then:
     beta: { "$eval": "BETA_APPLICATION_ID" }
     release: { "$eval": "RELEASE_APPLICATION_ID" }
+    esr: { "$eval": "ESR_APPLICATION_ID" }
```
```suggestion
    esr:
      $if: 'COT_PRODUCT == "thunderbird"
      then: { "$eval": "ESR_APPLICATION_ID" }
      else: null    
```

I think we just need to prevent the key from being calculated for firefox.

hneiva commented on pushmsixscript/docker.d/init_worker.sh:
```diff
@@ -25,7 +25,8 @@ case $COT_PRODUCT in
   thunderbird)
     export BETA_APPLICATION_ID=9PFQWVMSS45P
-    export RELEASE_APPLICATION_ID=9PM5VM1S3VMQ
+    export RELEASE_APPLICATION_ID=9MX7TGVGQFCL
```
I just noticed the `RELEASE_APPLICATION_ID` was also changed, is this intentional?

stephenking314 replied:
Yeah, this was intentional.   Our existing listing was always actually ESR.   We decided to keep that listing as ESR, label as such, and create a new listing for the standard release channel.

stephenking314 committed:
Fix Test Failures

Validator does not allow null value for 'esr' application ID
Using an empty string instead.

pre-commit-ci[bot] committed:
style: pre-commit.ci auto fixes [...]

hneiva reviewed (approved):
LGTM
